package.json

```diff
@@ -22,17 +22,17 @@
   },
   "scripts": {
     "fetch:style:spec": "./scripts/download-style-spec.sh",
-    "generate": "yarn node ./scripts/autogenerate",
-    "test": "npm run lint && npm run unittest",
+    "generate": "npx node ./scripts/autogenerate",
+    "test": "npm run lint && pnpm run unittest",
     "unittest": "jest",
     "unittest:single": "jest --testNamePattern",
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
-    "prepack": "pinst --disable && yarn build:plugin",
+    "prepack": "pinst --disable && pnpm build:plugin",
     "test:plugin": "jest plugin",
     "build:plugin": "tsc --build plugin",
     "lint:plugin": "eslint plugin/src/*",
-    "postinstall": "husky install",
+    "_postinstall": "husky install",
```

Comment:
Doing this in postinstall was a yarn 2+ workaround. Look here how to properly configure husky for pnpm https://typicode.github.io/husky/getting-started.html#automatic-recommended